examples/metalk8s-solution-example/config.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: solutions.metalk8s.scality.com/v1alpha1
+kind: SolutionConfig
```

Comment:
Would be cool to show everything we support in this file. Next PR!

Reply:
Yes, good idea.